src/language/templates/pybind/pynode.cpp

```diff
@@ -0,0 +1,89 @@
+/*************************************************************************
+ * Copyright (C) 2018-2021 Blue Brain Project
+ *
+ * This file is part of NMODL distributed under the terms of the GNU
+ * Lesser General Public License. See top-level LICENSE file for details.
+ *************************************************************************/
+
+///
+/// THIS FILE IS GENERATED AT BUILD TIME AND SHALL NOT BE EDITED.
```

Comment:
I don't think this should be committed - or am I missing something?

Reply:
Seems to be the precedent that we commit this text in the template files, e.g. https://github.com/BlueBrain/nmodl/blob/55e4a6f10c7843bdc1e8a2cb9d1193d2a3590a56/src/language/templates/pybind/pyast.cpp#L8-L10 too 🤷